pages/base/user.inc

```diff
@@ -14,13 +14,18 @@ if (empty($_SESSION['state']) && empty($_SESSION['accountInfo']['province'])) {
 ?>
 <?php if ($active) { ?>
 <h3>Your current profile information</h3>
-<form action="index.php?Function=profile" method="post" onsubmit="return validateForm(this)" name="Profile" id="profile_update" class="UI-container">
+<form onsubmit="updateAccount(<?php echo $_SESSION['accountId']?>);" name="Profile" id="profile_update" class="UI-container">
```

Comment:
I see what you mean about this file right now -- it's very "old school", isn't it :-)